cl/citations/utils.py

```diff
@@ -1,7 +1,14 @@
+from datetime import date
+from typing import Tuple
```

Comment:
We don't need this import. We can use the built-in `tuple`.

Reply:
Yeah, fixed it